src/MedTrackerAPI/Swagger/SwaggerExtension.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.OpenApi.Models;
+
+namespace MedTrackerAPI.Swagger;
+
+public static class SwaggerExtension
+{
+    public static void AddSwagger(this IServiceCollection services)
+    {
+        services.AddSwaggerGen(c =>
+        {
+            c.SwaggerDoc("v1", new OpenApiInfo
+            {
+                Title = "MedTracker API", Version = "v1"
+            });
+            c.AddSecurityDefinition("Bearer", new OpenApiSecurityScheme
+            {
+                In = ParameterLocation.Header,
+                Description = "Please provide a valid token",
+                Name = "Authorization",
+                Type = SecuritySchemeType.Http,
+                BearerFormat = "JWT",
+                Scheme = "Bearer",
+            });
+            c.AddSecurityRequirement(new OpenApiSecurityRequirement
+            {
+                {
+                    new OpenApiSecurityScheme
+                    {
+                        Reference = new OpenApiReference
+                        {
+                            Type = ReferenceType.SecurityScheme,
+                            Id = "Bearer"
+                        }
+                    },
+                    Array.Empty<string>()
```

Comment:
Can be simplified to []

Reply:
![image](https://github.com/user-attachments/assets/e204b5fc-a9cf-4e23-b238-969baa5b1677)